packages/loot-core/src/server/db/index.ts

```diff
@@ -454,10 +453,6 @@ export async function deletePayee(payee) {
   //   mappings.map(m => update('payee_mapping', { id: m.id, targetId: null }))
   // );
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏IMO worth deleting this too (but feel free to ignore for your PR as this is scope creep)

Reply:
Gonna hold off for now